From 73c0e985518c06e46cf90dae296c6939f11d6f19 Mon Sep 17 00:00:00 2001
From: Lars Greiss <kufikugel@googlemail.com>
Date: Sat, 4 Jan 2014 16:30:02 +0100
Subject: [PATCH 021/171] Camera2: some aapt warnings cleanup

during the move from Gallery2 into new Camera2 package some
unused values were moved as well. Remove them

Change-Id: I4a514c76ccc70c61bb0a97ecafda7e1f9ede9f35
---
 res/values-large/filtershow_values.xml  | 32 -------------------------
 res/values-xlarge/filtershow_values.xml | 32 -------------------------
 res/values-xlarge/styles.xml            |  4 ----
 3 files changed, 68 deletions(-)
 delete mode 100644 res/values-large/filtershow_values.xml
 delete mode 100644 res/values-xlarge/filtershow_values.xml

diff --git a/res/values-large/filtershow_values.xml b/res/values-large/filtershow_values.xml
deleted file mode 100644
index 33b8642ba..000000000
--- a/res/values-large/filtershow_values.xml
+++ /dev/null
@@ -1,32 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2013 The Android Open Source Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<resources>
-    <!-- Specify the screen orientation -->
-    <bool name="only_use_portrait">false</bool>
-
-    <!-- Category Panel Height -->
-    <dimen name="category_panel_height">94dip</dimen>
-
-    <!-- Category Panel Icon Size -->
-    <dimen name="category_panel_icon_size">72dip</dimen>
-
-    <!-- Category Panel Text Size -->
-    <dimen name="category_panel_text_size">14dip</dimen>
-
-    <!-- Category Panel Text Size -->
-    <dimen name="category_panel_margin">4dip</dimen>
-</resources>
\ No newline at end of file
diff --git a/res/values-xlarge/filtershow_values.xml b/res/values-xlarge/filtershow_values.xml
deleted file mode 100644
index c6fe3996b..000000000
--- a/res/values-xlarge/filtershow_values.xml
+++ /dev/null
@@ -1,32 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2013 The Android Open Source Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<resources>
-    <!-- Specify the screen orientation -->
-    <bool name="only_use_portrait">false</bool>
-
-    <!-- Category Panel Height -->
-    <dimen name="category_panel_height">106dip</dimen>
-
-    <!-- Category Panel Icon Size -->
-    <dimen name="category_panel_icon_size">84dip</dimen>
-
-    <!-- Category Panel Text Size -->
-    <dimen name="category_panel_text_size">14dip</dimen>
-
-    <!-- Category Panel Text Size -->
-    <dimen name="category_panel_margin">4dip</dimen>
-</resources>
\ No newline at end of file
diff --git a/res/values-xlarge/styles.xml b/res/values-xlarge/styles.xml
index 55b29b1fa..c413052b1 100644
--- a/res/values-xlarge/styles.xml
+++ b/res/values-xlarge/styles.xml
@@ -14,12 +14,8 @@
      limitations under the License.
 -->
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <style name="DialogPickerTheme" parent="android:Theme.Holo.Dialog">
-    </style>
-    <bool name="picker_is_dialog">true</bool>
 
     <!-- Camera resources below -->
-
     <style name="ReviewControlText" parent="@style/ReviewControlText_xlarge" />
     <style name="PopupTitleText" parent="@style/PopupTitleText_xlarge" />
     <style name="PanoCustomDialogText" parent="@style/PanoCustomDialogText_xlarge" />
-- 
2.17.1

